src/libraries/Pausable.sol

```diff
@@ -49,6 +84,21 @@ abstract contract Pausable is Initializable {
         _getPauseStorage()._pauseFlag = pauseFlag;
     }
 
+    function __Paused_init(address initialPauser, address owner) internal onlyInitializing {
```

Comment:
I don't think we're calling this initialiser from anywhere, or at least I can't see find it?

Reply:
its in now